first-analysis-steps/loki-functors.md

```diff
@@ -64,6 +65,15 @@ harmless in the examples we will use.
 If the import is made *before* the instantiation of the `ApplicationMgr`, there 
 will be no warnings.
 
+{% callout "A note about units" %}
+By convention, the LHCb default units are MeV, centimeters and picoseconds. It is easy to print the values of interest in other units:
```

Comment:
I thought the [convention for time was nanoseconds](http://lhcb-comp.web.cern.ch/lhcb-comp/Support/Conventions/units.pdf)?

Reply:
Indeed, it's my fault.

Reply:
No problem 😄 also I've just realised it should be millimeters instead of centimeters